### DISCUSSION TITLE
This user account is blocked ?????

### DISCUSSION THREAD

**Discussion by niruallen** [vercel/community#4453] [Account Issue]
### Question

hey, Vercel!
This is the second time I have encountered this issue; this is my second client's account being blocked right at the REGISTRATION process.

I have been waiting for your reply for more than 8 days now!!! 🤦‍♂️

I've been contacting (emailing) the registration and support team for more than a week now! 

Can you help me fix this, please? Please!

**Comment by swarnava** [comment]
Hi, Can you share Case #?

**Comment by Developsom** [comment]
i have the same problem :S
